apps/docs/docs/template/api/experimental/_category_.json

```diff
@@ -1,6 +1,6 @@
 {
   "label": "Experimental API",
-  "position": 4,
+  "position": 100,
```

Comment:
Why is this so high? 
If it is the only item in the list that requires being in order and it requires being last it can be placed in position 2.

Reply:
This is the last item.